frontend/docker-compose.yml

```diff
@@ -4,31 +4,33 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
-      args:
-        - NEXT_PUBLIC_API_HOST=https://www.resilienceatlas.org
-        - NEXT_PUBLIC_GOOGLE_ANALYTICS_ID=UA-123456789-1
-        - NEXT_PUBLIC_STATIC_JOURNEYS=true
     ports:
       - "3000:3000"
     container_name: resilienceatlas-frontend
-    command: node server
+    command: yarn start
+    env_file:
+      - .env.local
     environment:
       - PORT=3000
       - NODE_ENV=production
+      - NEXT_PUBLIC_GOOGLE_ANALYTICS_ID=UA-123456789-1
+      - NEXT_PUBLIC_STATIC_JOURNEYS=true
```

Comment:
Do you need to add `NEXT_PUBLIC_API_HOST=https://www.resilienceatlas.org` as well?